### PR TITLE
o1vm/pickles : uncombine constraints per instruction

### PR DESCRIPTION
**Pull request by marcbeunardeau88**

### CODE REVIEW

File: o1vm/src/pickles/main.rs [hunk 1]
```diff
@@ -83,17 +83,49 @@ pub fn main() -> ExitCode {
...
+                       Box::new(*small.get(i).unwrap_or(&E::<Fp>::zero())),
+                   ))
+                });*/
+                let res = big
+                    .into_iter()
+                    .enumerate()
+                    .map(|(i, cst)| cst + small.get(i).unwrap_or(&E::<Fp>::zero()).clone())
+                    .collect();
+
+                /*  // padd the acc or constraint to the same length
```
Comment (dannywillems) [inline o1vm/src/pickles/main.rs:113]: I guess it is not ready.

**Review by dannywillems — CHANGES_REQUESTED** [review]
See comment. Change PR state as draft.

File: o1vm/src/pickles/main.rs (hunk 1 above)
Comment (marcbeunardeau88) [reply o1vm/src/pickles/main.rs]: removed commented code

**Comment by dannywillems** [comment]
@marcbeunardeau88 can you fix the conflicts and rebase on master? We do have now the prover + e2e tests in the CI; I was waiting for this to get a second look here.

**Pull request closed without merging**